src/cursor/cursor.ts

```diff
@@ -39,9 +43,58 @@ export type CursorFlag = typeof FLAGS[number];
 /** @public */
 export const FIELDS = ['numberOfRetries', 'tailableRetryInterval'] as const;
 
+/** @public */
+export interface DocumentTransforms {
+  /** Transform each document returned */
+  doc(doc: Document): Document;
+  /** Transform the value returned from the initial query */
+  query?(doc: Document): Document | Document[];
+}
+
+/** @internal */
+export interface CoreCursorPrivate {
```

Comment:
two questions:
- can we just rename this `CursorPrivate`, it looks like its an alias below
- do we need to export this internal type?

Reply:
Currently both `CursorPrivate` and `CoreCursorPrivate` are [exported](https://github.com/mongodb/node-mongodb-native/blob/054838f00e75a1ed6e41f320c55d62fb3b63e583/src/index.ts#L149-L157) by `index.ts`, I wasn't sure why they were being exported or if I could safely fully delete them.

Reply:
yeah feel free to do that, I think this is because `api-extractor` complained about them not having a visibility tag, but we shouldn't be exporting internals like this